application/config.json.template

```diff
@@ -43,5 +43,6 @@
            "Other"
        ],
        "categoryRoleSuffix": " - Helper"
-   }
+   },
+    "memeChannelPattern": "memes"
```

Comment:
Personally feel like using ID would be wiser as there's only 1 channel

Reply:
or maybe we rename it to `mediaOnlyChannelPattern`. that way the system could be used for other channels as well. (we dont really have any other for now, but still)

Reply:
renamed to mediaOnlyChannelPattern